tests/integrate_test/cita_jsonrpcTest.sh

```diff
@@ -34,6 +34,11 @@ invalid_data=-32600
 invalid_jsonrpc_method=-32601
 
 ./cita_start.sh
+
```

Comment:
This script is not in CI.
Need another issue to check these scripts and add them back to CI.

Reply:
:+1:  Let me have a try.